GitUI/Editor/RebaseTodoHighlightingStrategy.cs

```diff
@@ -89,66 +89,66 @@ private static bool TryHighlightInteractiveRebaseCommand(IDocument document, Lin
                 switch (state)
                 {
                     case State.Command:
-                    {
-                        if (index == 1 && char.IsWhiteSpace(c))
                         {
-                            state = State.SpacesAfterCommand;
-                        }
-                        else if (index == cmd.longForm.Length && char.IsWhiteSpace(c))
-                        {
-                            state = State.SpacesAfterCommand;
-                        }
-                        else if (index >= cmd.longForm.Length || c != cmd.longForm[index])
-                        {
-                            return false;
-                        }
+                            if (index == 1 && char.IsWhiteSpace(c))
+                            {
```

Comment:
I guess, this additional indentation was not intended, was it?

Reply:
No it wasn't. Looks like my VS settings are different